src/esn.calendar.libs/app/services/event-service.js

```diff
@@ -535,6 +539,20 @@ function calEventService(
     });
   }
 
+  /**
+   * Change the status of participation from external link
+   * @param {Mixed}    eventUid   the uid of event
+   * @param {Mixed}    jwt        jwt
+   */
+  function changeParticipationFromLink(eventUid, jwt) {
+
+    return $http({ method: 'GET', url: getChangeParticipationUrl(jwt) })
```

Comment:
I think we should catch an error here and display a notification, something like "Something went wrong when changing your participation status for the event".